vars/patchDockerImage.groovy

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+void call(Map args = [:]) {
+    def lib = library(identifier: 'jenkins@main', retriever: legacySCM(scm))
+    String docker_image = "opensearchproject/${args.product}:${args.tag}"
+    String latest_docker_image = "opensearchproject/${args.product}:latest"
+    boolean tag_latest = false
+
+
+    sh """#!/bin/bash
+    set -e
+    set +x
+    docker pull ${docker_image}
+    docker pull ${latest_docker_image}
+    docker inspect --format '{{ index .Config.Labels "org.label-schema.version"}}' ${docker_image} > versionNumber
+    docker inspect --format '{{ index .Config.Labels "org.label-schema.build-date"}}' ${docker_image} > time
+    docker inspect --format '{{ index .Config.Labels "org.label-schema.description"}}' ${docker_image} > buildNumber
+    docker inspect --format '{{ index .Config.Labels "org.label-schema.version"}}' ${latest_docker_image} > latestVersionNumber
```

Comment:
I didnt realize you are creating files here.
You can simply use this to save all the values?
```
GIT_COMMIT_EMAIL = sh (
    script: 'git --no-pager show -s --format=\'%ae\'',
    returnStdout: true
).trim()
```

Possibly I miss it in the initial review?

Thanks.